packages/rsocket-core/src/RSocketMachine.js

```diff
@@ -485,11 +482,8 @@ class RSocketMachineImpl<D, M> implements RSocketMachine<D, M> {
                   },
                 });
               } else {
-                warning(
-                  false,
-                  'RSocketClient: re-entrant call to request n before initial' +
-                    ' channel established.',
-                );
+                console.warn('RSocketClient: re-entrant call to request n before initial' +
+                  ' channel established.');
```

Comment:
As far as I can tell, this line is the only non-equivalent replacement - previously it would only print a warning in the `_DEV_` mode. However, I feel like this situation deserves an actual always-on warning similar to the warning below (line 498).